materialize-sns/driver.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sts"
+
+	m "github.com/estuary/connectors/go/protocols/materialize"
+	boilerplate "github.com/estuary/connectors/materialize-boilerplate"
+	pf "github.com/estuary/flow/go/protocols/flow"
+	pm "github.com/estuary/flow/go/protocols/materialize"
+)
+type driver struct{}
+
+type config struct {
+	SNSTopicARN string `json:"snsTopicArn" jsonschema:"title=SNS Topic ARN,description=AWS SNS Topic ARN to publish messages to."`
```

Comment:
Materializations must support having multiple bindings, and I think each one would have a separate topic. It probably wouldn't have a direct ARN input; rather the name of a topic, and then it would have to manage creating topics that were needed and then looking up their ARNs.